examples/cluster/plot_kmedoids_digits.py

```diff
@@ -0,0 +1,81 @@
+"""
+=============================================================
+A demo of K-Medoids clustering on the handwritten digits data
+=============================================================
+
+In this example we compare different pairwise distance
+metrics for K-Medoids.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.cluster import KMedoids
+from sklearn.datasets import load_digits
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import scale
+from sklearn.metrics.pairwise import PAIRWISE_DISTANCE_FUNCTIONS
+
+print(__doc__)
+
+# Authors: Timo Erkkilä <timo.erkkila@gmail.com>
+#          Antti Lehmussola <antti.lehmussola@gmail.com>
+#          Kornel Kiełczewski <kornel.k@plusnet.pl>
+# License: BSD 3 clause
+
+np.random.seed(42)
+
+digits = load_digits()
+data = scale(digits.data)
+n_digits = len(np.unique(digits.target))
+
+reduced_data = PCA(n_components=2).fit_transform(data)
+
+# Step size of the mesh. Decrease to increase the quality of the VQ.
+h = .02     # point in the mesh [x_min, m_max]x[y_min, y_max].
+
+# Plot the decision boundary. For that, we will assign a color to each
+x_min, x_max = reduced_data[:, 0].min() - 1, reduced_data[:, 0].max() + 1
+y_min, y_max = reduced_data[:, 1].min() - 1, reduced_data[:, 1].max() + 1
+xx, yy = np.meshgrid(np.arange(x_min, x_max, h), np.arange(y_min, y_max, h))
+
+plt.figure()
+plt.clf()
+
+plt.suptitle("Plugging different distance functions to K-Medoids", fontsize=14)
+
+all_distance_metrics = PAIRWISE_DISTANCE_FUNCTIONS.keys()
+valid_distance_metrics = [d for d in all_distance_metrics if d
```

Comment:
looking at the example, there is exactly three metrics, but under 6 different names. Also, I'm not sure projecting the digits to 2d is the best example.